Elements/src/Geometry/Line.cs

```diff
@@ -519,6 +519,36 @@ public static bool PointOnLine(Vector3 point, Vector3 start, Vector3 end, bool i
             return false;
         }
 
+        /// <summary>
+        /// Calculates where the point is wrt to the line, assuming the point is on the actual line that goes through Start, End
+        /// </summary>
+        /// <param name="point">A point to be tested</param>
+        /// <returns></returns>
+        public int PointRelativePosition(Vector3 point)
```

Comment:
Adding new function for this is too much. Just use (point - start).Dot(end - start).